core/trino-main/src/test/java/io/trino/connector/MockConnector.java

```diff
@@ -591,25 +591,29 @@ public ConnectorTableProperties getTableProperties(ConnectorSession session, Con
         @Override
         public void grantSchemaPrivileges(ConnectorSession session, String schemaName, Set<Privilege> privileges, TrinoPrincipal grantee, boolean grantOption)
         {
-            accessControl.grantSchemaPrivileges(schemaName, privileges, grantee, grantOption);
+            accessControl.orElseThrow()
```

Comment:
`getAccessControl().grantSchemaPrivileges(schemaName, privileges, grantee, grantOption)`?

Reply:
I think the current version is more explicit

Reply:
Actually, you right Thanks